clicache/src/impl/PdxRemoteWriter.hpp

```diff
@@ -220,74 +220,74 @@ namespace Apache
           //virtual IPdxWriter^ writeFile(String fieldName, File file) ;
 
           /// <summary>
-          /// Write an collection to the <c>IPdxWriter</c>.
+          /// Write a boolean array to the <c>IPdxWriter</c>.
           /// </summary>
           /// <param name="fieldName">The name of the field associated with the value.</param>
           /// <param name="boolArray">The boolArray to write.</param>
 					virtual IPdxWriter^ WriteBooleanArray( String^ fieldName, array<Boolean>^ boolArray) override;
 
           /// <summary>
-          /// Write an collection to the <c>IPdxWriter</c>.
+          /// Write an char array to the <c>IPdxWriter</c>.
```

Comment:
What the heck is `Char` in this context?  8-bit?  16-bit?  Variable size?  Signed or unsigned?

Reply:
Char is technically neither signed or unsigned. It has its own rules regarding casts and how const chars are initialized.